core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -366,6 +366,18 @@ private[spark] class AppStatusStore(
     Some(computedQuantiles)
   }
 
+  /**
+   * Calculates a summary of the executor metrics for executors, returning the
+   * requested quantiles for the recorded metrics.
+   */
+  def executorMetricSummary(activeOnly: Boolean,
+                            unsortedQuantiles: Array[Double])
```

Comment:
indentation si off

Reply:
Thanks for review. Updated all based on your comments.